lingua_franca/lang/format_pl.py

```diff
@@ -0,0 +1,351 @@
+                q, r = divmod(n, 100)
+                digit_name = digits[q]
+                if q*100 in _NUM_STRING_PL:
+                    digit_name = _NUM_STRING_PL[q*100]
+
+                return digit_name + (
+                    " " + _sub_thousand(r, ordinals) if r else "")
+
+        def _short_scale(n):
+            if n >= max(_SHORT_SCALE_PL.keys()):
+                return "nieskończoność"
+            ordi = ordinals
+
+            if int(n) != n:
+                ordi = False
+            n = int(n)
+            assert 0 <= n
+            res = []
+            for i, z in enumerate(_split_by(n, 1000)):
+                if not z:
+                    continue
+                number = _sub_thousand(z, ordi, iteration=i)
+
+                if i:
+                    if i >= len(hundreds):
+                        return ""
+                    number += " "
+                    if ordi:
+                        if i * 1000 in _SHORT_ORDINAL_PL:
+                            if z == 1:
+                                number = _SHORT_ORDINAL_PL[i * 1000]
+                            else:
+                                number += _SHORT_ORDINAL_PL[i * 1000]
+                        else:
+                            if n not in _SHORT_SCALE_PL:
+                                num = int("1" + "0" * (len(str(n)) - 2))
+
+                                number += _SHORT_SCALE_PL[num] + "owa"
+                            else:
+                                number = _SHORT_SCALE_PL[n] + "ty"
+                    else:
+                        hundreds_text = _SHORT_SCALE_PL[float(pow(1000, i))]
+                        if z != 1:
+                            _, z_mod = divmod(z, 10)
+                            _, z_mod_tens = divmod(z, 100)
+                            n_main, _ = divmod(z_mod_tens, 10)
+                            if i == 1:
+                                if n_main != 1 and 5 > z_mod > 0:
+                                    hundreds_text += "e"
+                                else:
+                                    hundreds_text = "tysięcy"
+                            elif i > 1:
+                                hundreds_text += "y" if 5 > z_mod > 0 else "ów"
+
+                        number += hundreds_text
+                res.append(number)
+                ordi = False
+
+            return ", ".join(reversed(res))
+
+        def _split_by(n, split=1000):
+            assert 0 <= n
+            res = []
+            while n:
+                n, r = divmod(n, split)
+                res.append(r)
+            return res
+
+        result += _short_scale(num)
+
+    # deal with scientific notation unpronounceable as number
+    if not result and "e" in str(num):
+        return pronounce_number_pl(num, places, short_scale, scientific=True)
+    # Deal with fractional part
+    elif not num == int(num) and places > 0:
+        if abs(num) < 1.0 and (result is "minus " or not result):
```

Comment:
I apologize for failing to notice this before you rebased. This `is` raises a SyntaxWarning, fixed in the other files some time after you forked.

Easy fix, `if abs(num) < 1.0 and (result == "minus " or not result):`

`git commit --amend` should be okay, given that you're force-pushing regardless.